docs/source/quickstart/model_config.md

```diff
@@ -6,14 +6,23 @@ Currently, ml4ir supports a dense neural network architecture (multi layer perce
 
 **Note**: To train a simple linear ranking model, use the architecture_key as `linear` with a single `dense` layer.
 
-This file is also used to define the optimizer and the learning rate schedule. The current supported optimizers are: `adam`, `adagrad`, `nadam`, `sgd`, `rms_prop`. Each of these optimizers need so set the following hyper-parameter: `gradient_clip_value`. `adam` is the default optimizer if non was specified.
+This file is also used to define the optimizer, the learning rate schedule and calibration with
+ temperature scaling. The current
+ supported optimizers are: `adam`, `adagrad`, `nadam`, `sgd`, `rms_prop`. Each of these optimizers need so set the following hyper-parameter: `gradient_clip_value`. `adam` is the default optimizer if non was specified.
 The current supported learning rate schedules are: `exponential`, `cyclic` and `constant`. `constant` is the default schedule if non was specified with learning rate = 0.01
 
 The `exponential` learning rate schedule requires defining the following hyper-parameters: `initial_learning_rate`, `decay_steps`, `decay_rate`. For more information, see: https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/schedules/ExponentialDecay
 
 The `cyclic` learning rate schedule has three different type of policies: `triangular`, `triangular2`, `exponential`. All three types require defining the following hyper-parameters: `initial_learning_rate`, `maximal_learning_rate`, `step_size`. The `exponential` type requires and additional hyper-parameter: `gamma`. 
 For more information, see: https://www.tensorflow.org/addons/api_docs/python/tfa/optimizers/CyclicalLearningRate and https://arxiv.org/pdf/1506.01186.pdf.
 
+Calibration will be done as a separate process after possibly training or evaluating a
+ (classification) model. It implements [temperature scaling](https://github.com/gpleiss/temperature_scaling) technique to
+ calibrate output probabilities of a classifier. It uses the `validation` set to train a
+  `temperature` parameter, defined in the `ModelConfig` file. Then, it evaluates the calibrated
+   model
+   on the `test` set and stores the probability scores before and after applying calibration.
```

Comment:
@marjanhs Worth mentioning that we currently don't save a calibrated model under "Limitations". Maybe even call out RankingModel is not supported.

This looks good otherwise. Thank you :)